docs/how-to/deploy_sdcore.md

```diff
@@ -2,9 +2,15 @@
 
 This guide explains how to deploy the SD-Core charm bundle.
 
+## SD-Core bundle
+
+This bundle includes the Control Plane Function charms, the User Plane Function (UPF), the WebUI for configuration and the Grafana Agent to integrate with the Observability stack.
+
 ## Requirements
 
 - Juju >= 3.1
+> **Note**
+> In case of using MicroK8s be aware that Juju 3 requires a strictly confined MicroK8s version.
```

Comment:
I don't want us to maintain information that is not SD-Core/Charmed 5G specific. MicroK8s is one of the K8s options but it is not a pre-requisites. For instance we're not placing caviats on the EKS, GKE, ... other k8s deployments.

In other words, I'd remove this note.

In the tutorial we'll use Microk8s and it will be clear that this needs to be set.